### PR TITLE
Added automatic_speech_recognition example

### PR DESCRIPTION
**Pull request by y27choi**
### What change does this PR introduce and why?
Added ASR example
Using kolena.metrics in semantic segmentation example

### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

[example on trunk/yoohee](https://trunk.kolena.io/yoohee/dataset?datasetId=31)

### CODE REVIEW

File: examples/dataset/automatic_speech_recognition/README.md
Comment (gordonhart) [inline examples/dataset/automatic_speech_recognition/README.md:1]: Should also update the `examples/dataset/README.md` with a bullet point for this example.

File: examples/dataset/automatic_speech_recognition/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+4. Failure Rate (`rate(is_failure=true)`)
+
+### Plots
+
+1. `mean(result.character_error_rate)` vs. `datapoint.duration_seconds`
+2. `mean(result.character_error_rate)` vs. `datapoint.tempo`
+
+### Test Cases
+
+1. `datapoint.speaker_sex`
```
Comment (gordonhart) [inline examples/dataset/automatic_speech_recognition/README.md:83]: [Minor] this is propagated from the `kolena.workflows` ASR example but the `speaker_sex` metadata values are still padded with spaces:

Screenshot 2024-01-31 at 9 01 16 AM

File: examples/dataset/automatic_speech_recognition/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --dataset DATASET     Optionally specify a custom dataset name to test.
+```
+
+## Quality Standards Guide
+
+Once the dataset and results have been uploaded to Kolena, visit [Kolena](https://app.kolena.io/redirect/) to
+test the rain forecast models. See our [QuickStart](https://docs.kolena.io/dataset/quickstart/) guide
```
Comment (gordonhart) [inline examples/dataset/automatic_speech_recognition/README.md]: > rain forecast models

Update task name

File: examples/dataset/automatic_speech_recognition/README.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+for details.
+
+Here are our [Quality Standards](https://docs.kolena.io/dataset/core-concepts/quality-standard/) recommendations for
+this workflow:
+
+### Metrics
+
+1. [Word Error Rate](https://docs.kolena.io/metrics/wer-cer-mer)
+2. [Character Error Rate](https://docs.kolena.io/metrics/wer-cer-mer/)
+3. [Match Error Rate](https://docs.kolena.io/metrics/wer-cer-mer/)
```
Comment (gordonhart) [inline examples/dataset/automatic_speech_recognition/README.md]: Should we specify e.g. `mean(word_error_rate)`?

Screenshot 2024-01-31 at 9 06 26 AM

**Review by gordonhart — APPROVED** [review]
Looks great!

Screenshot 2024-01-31 at 9 08 38 AM

File: examples/dataset/automatic_speech_recognition/README.md (hunk 1 above)
Comment (y27choi) [reply examples/dataset/automatic_speech_recognition/README.md:83]: Updated

**Pull request merged**